pkg/model/manifest.go

```diff
@@ -1178,3 +1178,12 @@ func (d ManifestDevs) HasDev(name string) bool {
 	_, ok := d[name]
 	return ok
 }
+
+// GetDevs returns a list of strings with the keys of devs defined
+func (d ManifestDevs) GetDevs() []string {
+	devs := make([]string, 0)
```

Comment:
Minor thing. If you know the length of the slice (in this case would be `len(d)`), it is better to create the slice with that size

Reply:
i've finally transformed to `[]string{}` and append the keys